stingray/pulse/modeling.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+from astropy.modeling import models, fitting
+
+
+def sinc(x):
```

Comment:
Could this be replaced by the `numpy.sinc` function? I feel like the numpy implementation might be faster, if it's the same function.